zipkin-web/src/main/resources/templates/v2/index.mustache

```diff
@@ -22,7 +22,7 @@
       <label for="timeStamp">End time</label>
       <input class="form-control input-sm date-input" type="text">
       <input class="form-control input-sm time-input" type="text">
-      <input class="" id="endTs" name="endTs" type="hidden" value="{{endTs}}">
+      <input class="timestamp-value" id="endTs" name="endTs" type="hidden" value="{{endTs}}">
```

Comment:
Does it mean to find by element ID in timestamp.js?
I think it is too specific and no other element like start time field can use timestamp class.